app/api/orders.py

```diff
@@ -675,11 +674,11 @@ def before_get(self, args, kwargs):
         """
         if kwargs.get('order_identifier'):
             order = safe_query(
-                db, Order, 'identifier', kwargs['order_identifier'], 'order_identifier'
+                Order, 'identifier', kwargs['order_identifier'], 'order_identifier'
             )
             kwargs['id'] = order.id
         elif kwargs.get('id'):
-            order = safe_query(db, Order, 'id', kwargs['id'], 'id')
+            order = safe_query(Order, 'id', kwargs['id'], 'id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)